docs/src/testnets/internal-testnet-1/README.md

```diff
@@ -1,35 +1,78 @@
 # Internal Testnet 1
 
-*Last updated on **3/30/2022** by **Alexandre Roque***
+##### Last updated on 04/07/2022 by Alexandre Roque
```

Comment:
this is in git, maybe we should add links to the source on github instead - I found https://github.com/badboy/mdbook-open-on-gh/, I'll add it in another PR

Reply:
That would be cool to have it. Also another cool add-on would be to have admonishments like https://github.com/tommilligan/mdbook-admonish. Useful to signal additional infos / warnings about the docs.

Reply:
alright, that looks great!

Reply:
Created PR #1036 for this 🙌